css/about.css

```diff
@@ -0,0 +1,167 @@
+
+#hero h2 {
+  margin-left: 50px;
+}
+
+#hero h3 {
+  margin: 5px 50px;
+  color: #ec5242;
+  text-overflow: inherit;
+}
+
+.date-place h4 {
+  font-size: 2rem;
+  margin-left: 20px;
+}
+
+.date-place a {
+  color: black;
+  border-bottom: 1px solid black;
+}
+
+#ourlogo {
+  height: 60vh;
+  padding: 20px;
+}
+
+#ourlogo .container {
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [ ] Use`flex-box` to align all items in `ourlogo` section in column. 
![image](https://user-images.githubusercontent.com/64894927/128204897-e058ee5c-a297-4dc6-ae1a-fd10b158bd70.png)
![image](https://user-images.githubusercontent.com/64894927/128204947-c394f550-3c0c-4cd8-8151-90f422187be9.png)

Reply:
Done thanks for your review 👍